### ISSUE TITLE
Rightklick ability cause endermite (namnetaged) despawn.

### ISSUE THREAD

**Issue by broken1arrow** [mcMMO-Dev/mcMMO-Classic#42]
So long i not use Rightklick ability, so not despawn the endermite (i can stand and kill endermans in 30 min no problem). 

But if i use Rightklick ability on endermans, he (endermite) disappear.

Is not only on server, is an other server some have same issue.

**Comment by t00thpick1** [comment]
I'm sorry could you rephrase your problem?

**Comment by broken1arrow** [comment]
Also if a use Serrated Strikes ore other mcmmo relate attacks on mobs, the endermite despawn (disappear). 

I have endermite 3 blocks away and a roof. he sitting in a minecart.

I use endermite in my farm for endermans.
